app/services/foreman/renderer/scope/macros/loaders.rb

```diff
@@ -45,6 +45,17 @@ module Loaders
               example "<% #{name}.each_record do |#{model.to_s.downcase}| %>
   <%= #{model.to_s.downcase}.id %>, <%= #{model.to_s.downcase}.#{model == User ? 'login' : 'name'} %>
 <% end %>", desc: "Prints #{model.to_s.downcase} id and #{model == User ? 'login' : 'name'} of each #{model.to_s.downcase}"
+              example "# Joins - for comparing and filtering data
```

Comment:
Would it be good to use heredoc?
```suggestion
              example <<~EXAMPLE
                # Joins - for comparing and filtering data
```

Or at least single quotes so you don't have to escape `"` inline.